__tests__/server-only.test-checkout-symlink-leading-path.js

```
@@ -0,0 +1,72 @@
+
+import { makeFixture } from './__helpers__/FixtureFS.js'
+
+// checkout should not write a file whose parent path traverses a symlink. git does not
+// follow symlinks in the leading path when writing working-tree files; this checks the
+// same. (node-only: needs real symlinks.)
+describe('checkout symlinked leading path', () => {
+  it('does not write through a planted directory symlink', async () => {
+    const { fs, dir, gitdir } = await makeFixture(
+      'test-checkout-symlink-leading-path'
+    )
+    const author = { name: 't', email: 't@t', timestamp: 1000, timezoneOffset: 0 }
```

ESLint

[error] Replace `·name:·'t',·email:·'t@t',·timestamp:·1000,·timezoneOffset:·0` with `⏎······name:·'t',⏎······email:·'t@t',⏎······timestamp:·1000,⏎······timezoneOffset:·0,⏎···`

(prettier/prettier)